core/tauri/src/menu/plugin.rs

```diff
@@ -877,7 +877,7 @@ pub(crate) fn init<R: Runtime>() -> TauriPlugin<R> {
     .on_event(|app, e| {
       if let RunEvent::MenuEvent(e) = e {
         if let Some(channel) = app.state::<MenuChannels>().0.lock().unwrap().get(&e.id) {
-          let _ = channel.send(&e.id);
+          let _ = channel.send(e.id.clone());
```

Comment:
shouldn't we allow sending a reference?

Reply:
I did consider that but `send` immediately serializes the value, so the string would have been cloned anyway. At least this way you can serialize owned values unlike if only references were supported, but maybe there's a way to support both i haven't considered